dataframe-geo/src/main/kotlin/org/jetbrains/kotlinx/dataframe/geo/geotools/toSimpleFeatureCollection.kt

```diff
@@ -0,0 +1,46 @@
+package org.jetbrains.kotlinx.dataframe.geo.geotools
+
+import org.geotools.api.feature.simple.SimpleFeature
+import org.geotools.data.collection.ListFeatureCollection
+import org.geotools.data.simple.SimpleFeatureCollection
+import org.geotools.feature.simple.SimpleFeatureBuilder
+import org.geotools.feature.simple.SimpleFeatureTypeBuilder
+import org.jetbrains.kotlinx.dataframe.api.forEach
+import org.jetbrains.kotlinx.dataframe.api.map
+import org.jetbrains.kotlinx.dataframe.api.single
+import org.jetbrains.kotlinx.dataframe.geo.GeoDataFrame
+import org.locationtech.jts.geom.Geometry
+
+fun GeoDataFrame<*>.toSimpleFeatureCollection(
+    name: String? = null,
+    singleGeometryType: Boolean = false
+): SimpleFeatureCollection {
+    val typeBuilder = SimpleFeatureTypeBuilder()
+    typeBuilder.name = name ?: "geodata"
+    typeBuilder.setCRS(crs)
+    val geometryClass = if (singleGeometryType) {
+        // todo singleOrNull() ?: error()
```

Comment:
many todo's haha

Reply:
I believe it's ok since it's very alpha.